glide.lock

```diff
@@ -203,4 +226,14 @@ imports:
   - util/integer
 - name: k8s.io/kubernetes
   version: df7f4b3526a580ef122aaeef61ab52842a60a88b
-testImports: []
```

Comment:
I ran `glide update -v` to update glide.lock. I can also run `glide update -v --skip-test` if that's preferred, but it's probably wise to also vendor test dependencies.